src/lib/test-setup.js

```diff
@@ -9,6 +9,9 @@ require("babel-register")({
   plugins: ["transform-object-rest-spread"]
 });
 
+require.extensions['.svg'] = () => null;
```

Comment:
Oh hey, I forgot about mocking require extensions like this - good catch!

Reply:
Interesting! I've never seen `require.extensions`.
I wasn't sure where/how we were using it below, but it looks like it may have been [deprecated since Node v0.10.6](https://nodejs.org/api/modules.html#modules_require_extensions).